task/git-rebase/0.1/git-rebase.yaml

```diff
@@ -82,7 +82,7 @@ spec:
   steps:
     - name: rebase
       workingDir: $(workspaces.source.path)
-      image: docker.io/alpine/git:v2.26.2@sha256:23618034b0be9205d9cc0846eb711b12ba4c9b468efdd8a59aac1d7b1a23363f #tag: v2.26.2
+      image: cgr.dev/chainguard/git
```

Comment:
can we add tag with this image? Also will it be possible to add a digest for this?

Reply:
Done.

Is there a dependabot like service that can be set up for the catalog?

I didn't do this originally because the last time this image was updated was 2 years ago. Using latest but keeping up to date with security fixes seemed like a better tradeoff then letting catalog tasks sit with images with known vulnerabilities.

Having automation bump the version automatically would help with this.

Reply:
not really sure on that @wlynch but we can try to setup one on `github.com/tektoncd-catalog` repository

Reply:
@wlynch renovate might work, but we do not have any service like that yet (planning to work on this but..).